.travis.yml

```diff
@@ -18,6 +18,7 @@ env:
   - DB=mysqli
 
 before_install:
+  - sudo sh -c "echo 'precedence ::ffff:0:0/96 100' >> /etc/gai.conf"
```

Comment:
I'm not sure how else to prevent composer from using ipv6 to update itself... We could maybe use wget to download composer.phar, but that's even uglier I think... Docker builds would be quite nice.